webapp/src/codecard.tsx

```diff
@@ -136,7 +136,7 @@ export class CodeCardView extends data.Component<CodeCardProps, CodeCardState> {
                     }
                 </div> : undefined}
             {card.time ? <div className="meta">
-                {card.tutorialLength ? <span className={`ui tutorial-progress ${tutorialDone ? "green" : "orange"} left floated label`}><i className={`${tutorialDone ? "trophy" : "circle"} icon`}></i>&nbsp;{lf("{0}/{1}", (card.tutorialStep || 0) + 1, card.tutorialLength)}</span> : undefined}
+                {card.tutorialLength ? <span className={`ui tutorial-progress ${tutorialDone ? "green" : "purple"} left floated label`}><i className={`${tutorialDone ? "trophy" : "circle"} icon`}></i>&nbsp;{lf("{0}/{1}", (card.tutorialStep || 0) + 1, card.tutorialLength)}</span> : undefined}
```

Comment:
I'm trying to figure out where this component is. A little surprised we had it hard-coded to orange before and not theme-specific. Is it only on skillmaps?

Reply:
This is the progress indicator in the cards on the home page. 
<img width="147" alt="tutorial-progress" src="https://github.com/microsoft/pxt/assets/49178322/d028a5d2-7f9b-4a5d-b76a-ba449e69b773">

I was also shocked that this was hard coded to orange. It was just handled in micro:bit to have a style override for the .orange class and set the color to purple. It looks like Minecraft does something a bit different and doesn't rely on the class being orange to set the style. I looked into accessing the "secondaryColor" less variable and then plugging the calculated color into the class here, but I couldn't find a nice way to do it, so I just put purple here instead.